### ISSUE TITLE
isNotNull filter doesn't seem to work

### ISSUE THREAD

**Issue by tylerbecks** [prisma/studio#1104]
### Bug description

I have null values, but I can't get the isNotNull filter to work.

### How to reproduce

1. `npx prisma studio`
2. Click Filters
3. Click Add a new filter
4. Select a nullable column
5. Filter by `isNotNull`
6. See null values

### Prisma information

generator client {
  provider = "prisma-client-js"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model Article {
  id          Int                     @id @default(autoincrement())
  createdAt   DateTime                @default(now()) @map("created_at") @db.Timestamptz(6)
  updatedAt   DateTime?               @map("updated_at") @db.Timestamptz(6)
  deletedAt   DateTime?               @map("deleted_at") @db.Timestamptz(6)
  url         String                  @unique(map: "unique_url") @db.VarChar
  title       String                  @db.VarChar
  source      String                  @map("article_source") @db.VarChar
  cityId      Int                     @map("city_id")
  description String?                 @db.VarChar
  imageUrl    String?                 @map("image_url") @db.VarChar
  restaurants ArticlesToRestaurants[] @relation("articlestorestaurantsToarticles")
  cities      City                    @relation("articlesTocities", fields: [cityId], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "fk_city_id")

  @@map("articles")
}

model Award {
  id           Int        @id @default(autoincrement())
  createdAt    DateTime   @default(now()) @map("created_at") @db.Timestamptz(6)
  updatedAt    DateTime?  @map("updated_at") @db.Timestamptz(6)
  deletedAt    DateTime?  @map("deleted_at") @db.Timestamptz(6)
  restaurantId Int        @map("restaurant_id")
  source       String     @map("award_source") @db.VarChar
  type         String     @map("award_type") @db.VarChar
  year         Int
  url          String     @db.VarChar
  chef         String?    @db.VarChar
  restaurants  Restaurant @relation(fields: [restaurantId], references: [id], onDelete: NoAction, onUpdate: NoAction)

  @@map("awards")
}

model City {
  city        String       @db.VarChar
  state       String?      @db.VarChar
  country     String       @db.VarChar
  id          Int          @id @default(autoincrement())
  articles    Article[]    @relation("articlesTocities")
  restaurants Restaurant[] @relation("citiesTorestaurants")

  @@unique([city, state, country])
  @@map("cities")
}

model Restaurant {
  id        Int                     @id @default(autoincrement())
  createdAt DateTime                @default(now()) @map("created_at") @db.Timestamptz(6)
  updatedAt DateTime?               @map("updated_at") @db.Timestamptz(6)
  deletedAt DateTime?               @map("deleted_at") @db.Timestamptz(6)
  name      String                  @db.VarChar
  gPlaceId  String?                 @unique @map("g_place_id") @db.VarChar
  cityId    Int                     @map("city_id")
  cuisine   String?                 @db.VarChar
  articles  ArticlesToRestaurants[] @relation("articlestorestaurantsTorestaurants")
  awards    Award[]
  cities    City                    @relation("citiesTorestaurants", fields: [cityId], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "fk_city_id")

  @@map("restaurants")
}

model ArticlesToRestaurants {
  createdAt    DateTime   @default(now()) @map("created_at") @db.Timestamptz(6)
  updatedAt    DateTime?  @map("updated_at") @db.Timestamptz(6)
  deletedAt    DateTime?  @map("deleted_at") @db.Timestamptz(6)
  articleId    Int        @map("article_id")
  restaurantId Int        @map("restaurant_id")
  description  String?    @db.VarChar
  id           Int        @id @default(dbgenerated("nextval('_articlestorestaurants_id_seq'::regclass)"))
  articles     Article    @relation("articlestorestaurantsToarticles", fields: [articleId], references: [id], onDelete: Cascade, onUpdate: NoAction, map: "articles_restaurants_article_id_fkey")
  restaurants  Restaurant @relation("articlestorestaurantsTorestaurants", fields: [restaurantId], references: [id], onDelete: Cascade, onUpdate: NoAction, map: "articles_restaurants_restaurant_id_fkey")

  @@map("_articlestorestaurants")
}

model PlaceIdCache {
  id        Int       @id @default(autoincrement())
  createdAt DateTime  @default(now()) @map("created_at") @db.Timestamptz(6)
  updatedAt DateTime? @map("updated_at") @db.Timestamptz(6)
  deletedAt DateTime? @map("deleted_at") @db.Timestamptz(6)
  placeId   String    @map("place_id") @db.VarChar
  name      String    @db.VarChar
  cityId    Int       @map("city_id")

  @@unique([name, cityId])
  @@map("place_id_cache")
}

### Environment & setup

No

### Screenshot of page where the problem happened (if applicable)

<img width="1728" alt="Screenshot 2023-03-20 at 9 00 17 PM" src="https://user-images.githubusercontent.com/16826230/226514674-097b08ca-24a7-439a-9ced-d1ae9e24950f.png">

**Comment by goleary** [comment]
ditto. this makes prisma studio pretty difficult to use

**Comment by alex-zwingli** [comment]
+1 This is a basic but impactful bug. Please fix ASAP.

**Comment by neinja007** [comment]
I am pretty sure that prisma studio simply ignores the filters because it detects that the filters are incorrect (as indicated by the red border next to the isNotNull / isNull option of the input box **that shouldnt even be there**)

**Comment by arbass22** [comment]
Seeing this issue as well with no workaround

**Comment by goleary** [comment]
yeah...kind of mind boggling that this has been broken for a year and a half...drizzle here i come.

**Comment by igalklebanov** [comment]
Hey :wave:

Prisma ORM 7 introduces a brand new standalone [Studio](https://www.prisma.io/docs/orm/tools/prisma-studio), built from the ground up to work everywhere, with or without Prisma ORM. Consequently, all "old Studio" issues are being closed as there won't be any further work on it. :mega:

The new Studio is also available in the Prisma VS Code extension and in the Prisma Console application. :electric_plug:

Help shape, improve and fix the new Studio with us in this repository by submitting bug reports and feature requests. :construction_worker:
